editor/src/document/overlays_message.rs

```diff
@@ -10,6 +10,7 @@ use serde::{Deserialize, Serialize};
 pub enum OverlaysMessage {
 	ClearAllOverlays,
 	DispatchOperation(Box<DocumentOperation>),
+	Redraw,
```

Comment:
We are not redrawing as that would involve the overlay document being cleared and all of the tools resubmitting their draw calls. Perhaps calling it `Rerender` or similar would be more informative?